resources/odvd/ODVDFH16Truck.odvd

```diff
@@ -16,11 +16,12 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-message opendlv.proxy.reverefh16.ManualControl [id = 191] {
+message opendlv.proxy.reverefh16.ManualDriver [id = 191] {
```

Comment:
This PR is connected to https://github.com/chalmers-revere/opendlv/pull/119 . 
Approving both should fix the inter-dependencies among the two repositories.